src/components/button/button.scss

```diff
@@ -109,6 +109,20 @@ button {
         }
     }
 
+    &.loading-failed {
+        limel-icon,
+        .label {
+            opacity: 0;
+        }
+        svg {
+            opacity: 1;
+            color: red;
+        }
+        limel-spinner {
+            display: block;
+        }
+    }
```

Comment:
Todo for @Kiarokh and @LucyChyzhova
We also need to render another svg for the loading failed state.